src/components/Register.vue

```diff
@@ -81,8 +88,10 @@ export default {
                 for (const param in uri_params) {
                     url.searchParams.set(param, uri_params[param])
                 }
-
-                window.location.href = url;
+                setTimeout(() => {
```

Comment:
What does this do? Why not abstract this to a function or an action instead of duplicating code?

Reply:
https://router.vuejs.org/guide/advanced/navigation-guards.html#optional-third-argument-next